resources/dicts/patrols/beach/hunting/greenleaf.json

```diff
@@ -1504,5 +1504,2300 @@
+                        "values": ["platonic", "comfort"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "r_c jokes that {PRONOUN/r_c/subject} can't possibly be related to s_c - clearly s_c was born from reef sharks, not cats, with the way {PRONOUN/s_c/subject} {VERB/s_c/glide/glides} through the water.",
+                "exp": 20,
+                "weight": 20,
+                "stat_skill": ["SWIMMER,1"],
+                "prey": ["medium"],
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "comfort"],
+                        "amount": 5
+                    }
+                ]
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "Sometimes, the prey just doesn't run, and today is one of those times.",
+                "exp": 0,
+                "weight": 20,
+                "prey": ["very_small"]
+            },
+            {
+                "text": "r_c sees a fish {PRONOUN/r_c/subject}'d like for the fresh-kill pile, but while perched on a rock setting up a good pounce a rogue wave washes them into the ocean - {PRONOUN/r_c/subject} {VERB/r_c/bob/bobs} to the surface and splutter {PRONOUN/r_c/poss} way to shore, but {PRONOUN/r_c/poss} breath comes short and wheezing after the dunking. Fleas and ticks!",
+                "exp": 0,
+                "weight": 10,
+                "injury": [
+                    {
+                        "cats": ["r_c"],
+                        "injuries": ["water in their lungs"],
+                        "scars": []
+                    }
+                ],
+                "prey": ["very_small"]
+            }
+        ]
+    },
+    {
+        "patrol_id": "bch_hunt_random_greenleaf_siblings3to6warriorslocked4",
+        "biome": ["beach"],
+        "season": ["greenleaf"],
+        "types": ["hunting"],
+        "tags": [],
+        "patrol_art": "hunt_general_intro",
+        "min_cats": 3,
+        "max_cats": 6,
+        "min_max_status": {
+            "all apprentices": [-1, -1]
+        },
+        "weight": 40,
+        "intro_text": "The hunting patrol heads out early, before the sun can turn the dunes into the bakingly hot expanse they've been recently.",
+        "decline_text": "They encounter a dawn patrol on their way back to camp and join up with them instead.",
+        "chance_of_success": 50,
+        "relationship_constraint": ["siblings"],
+        "success_outcomes": [
+            {
+                "text": "p_l grabs a massive b_mp_dl_s that {PRONOUN/p_l/subject} can barely carry, and r_c congratulates {PRONOUN/p_l/object}, secretly wondering if {PRONOUN/r_c/subject}'ll ever manage to surpass {PRONOUN/r_c/poss} sibling's achievements.",
```

Comment:
Would it be worthwhile for this outcome variation to be a hunter stat cat?

Reply:
All patrols need at least one normal non-trait or skill locked success still <3